CultivateMVP/Models/JournalOptionsProvider.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import RxSwift
+import RxCocoa
+
+class JournalOptionsProvider {
```

Comment:
And it shouldn't be in database it rather should be hardcoded.
Probably you want something like this:
```swift
struct JournalQuestion {
enum AnswerType {
case slider
case text
}
     let question: String
    let answerType: AnswerType
}
```
then you can pass default setup via init to your view model